packages/mobile/src/store/purchase-vendor/slice.ts

```diff
@@ -0,0 +1,26 @@
+import { PurchaseVendor } from '@audius/common'
```

Comment:
could this be state local to the purchase form instead?

Reply:
yeah i agree -- why does this have to be in redux at all?

Reply:
ohh bc the vendor-picker drawer is a separate component